controllers/octaviaapi_controller.go

```diff
@@ -984,6 +984,7 @@ func (r *OctaviaAPIReconciler) generateServiceSecrets(
 		endptConfig := map[string]interface{}{}
 		endptConfig["ServerName"] = fmt.Sprintf("%s-%s.%s.svc", octavia.ServiceName, endpt.String(), instance.Namespace)
 		endptConfig["TLS"] = false // default TLS to false, and set it bellow to true if enabled
+		endptConfig["TimeOut"] = instance.Spec.APITimeout
```

Comment:
we don't have timeout per endpoint, I don't think it's worth it that we set it there, we can set it directly in templateParameters (templateParameters["TimeOut"] = instance.Spec.APITimeout)
and use {{ TimeOut }} in the template